### PR TITLE
Adding axis2.xml properties and other carbon server and catalina related configs

### PR DESCRIPTION
**Pull request by CrowleyRajapakse**
## Purpose
Fixes 
https://github.com/wso2/docs-apim/issues/1933
https://github.com/wso2/docs-apim/issues/2018
https://github.com/wso2/docs-apim/issues/497
https://github.com/wso2/docs-apim/issues/1175

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by isharac — APPROVED** [review]

File: en/tools/config-catalog-generator/data/configs.json [hunk 1]
```diff
@@ -1761,6 +1761,1416 @@
                 }
             ],
             "exampleFile": "indexing.toml"
-        }                
+        },
+        {
+            "title": "HTTP Transport (catalina-server)",
```
Comment (dushaniw) [inline en/tools/config-catalog-generator/data/configs.json:1766]: Can we call it `HTTP Servlet Transport` ?

File: en/tools/config-catalog-generator/data/configs.json [hunk 2]
```diff
@@ -1761,6 +1761,1416 @@
...
+                            "possible": "-",
+                            "description": ""
+                        }
+                    ]
+                }
+            ],
+            "exampleFile": "_http-transport.toml"
+        },
+        {
+            "title": "HTTPS Transport (catalina-server)",
```
Comment (dushaniw) [inline en/tools/config-catalog-generator/data/configs.json:1915]: Similarly may be `HTTPS Servlet Transport`?

File: en/tools/config-catalog-generator/data/configs.json (hunk 1 above)
Comment (CrowleyRajapakse) [reply en/tools/config-catalog-generator/data/configs.json:1766]: will change

File: en/tools/config-catalog-generator/data/configs.json (hunk 2 above)
Comment (CrowleyRajapakse) [reply en/tools/config-catalog-generator/data/configs.json:1915]: will change

**Review by dushaniw — APPROVED** [review]

File: en/tools/config-catalog-generator/data/configs.json (hunk 1 above)
Comment (CrowleyRajapakse) [reply en/tools/config-catalog-generator/data/configs.json]: resolved

File: en/tools/config-catalog-generator/data/configs.json (hunk 2 above)
Comment (CrowleyRajapakse) [reply en/tools/config-catalog-generator/data/configs.json]: resolved

**Pull request merged**